### ISSUE TITLE
api_ca does not default system cert bundle

### ISSUE THREAD

**Issue by nlvw** [ansible-community/ara#468]
## What is the issue ?
When using ARA 1.6.0 with a HTTPS API Server the callback plugin will fail due to a certificate error.  It seems like it is not checking the default cert bundle on they system (Fedora 37 in my case) as the following fixes the error

```ini
[ara]
api_client = http
api_server = https://ara.example.com
api_ca = /etc/pki/tls/cert.pem
```

Without `api_ca` the connection fails with

```output
[WARNING]: Failure using method (v2_playbook_on_start) in callback plugin
(<ansible.plugins.callback.ara_default.CallbackModule object at 0x7f4b47a812d0>):
HTTPSConnectionPool(host='ara.nmsu.edu', port=443): Max retries exceeded with url:
/api/v1/playbooks (Caused by SSLError(SSLCertVerificationError(1, '[SSL:
CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer
certificate (_ssl.c:992)')))

PLAY [Test Ansible Connection] ************************************************************
[WARNING]: Failure using method (v2_playbook_on_play_start) in callback plugin
(<ansible.plugins.callback.ara_default.CallbackModule object at 0x7f4b47a812d0>):
'NoneType' object is not subscriptable

TASK [connection test] ********************************************************************
[WARNING]: Failure using method (v2_playbook_on_task_start) in callback plugin
(<ansible.plugins.callback.ara_default.CallbackModule object at 0x7f4b47a812d0>):
'NoneType' object is not subscriptable
ok: [hpc-wiki-p]

TASK [stop ssh client persistant connection] **********************************************

PLAY RECAP ********************************************************************************
hpc-wiki-p                 : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

[WARNING]: Failure using method (v2_playbook_on_stats) in callback plugin
(<ansible.plugins.callback.ara_default.CallbackModule object at 0x7f4b47a812d0>):
'NoneType' object is not subscriptable
```

## What should be happening ?
The default ca bundle trusted by the system should be used by default when `api_ca` is not specified

## Software Versions
Ansible = 7.1.0 (Core = 2.14)
OS = Fedora 37
Python = 3.11
ARA = 1.6.0
Install Source = pip

**Comment by hille721** [comment]
@dmsimard I guess in your case it's not a self signed certificate. As soon as you are using a self sign certificate you have to give the path to the system ca bundle to the Python requests library, e.g. by simple setting REQUESTS_CA_BUNDLE.
Have to do the same for my Ara installations.

**Comment by hille721** [comment]
And the path differ between e.g. Ubuntu and Fedora. Thus hard to set a default there.

**Comment by dmsimard** [comment]
@hille721 I don't have bandwidth to try and reproduce right now but we have a working integration test to validate that it works for self signed certificates here: https://github.com/ansible-community/ara-collection/pull/57/files and it looks sufficient to specify api_ca.

It hasn't run in a while but I will trigger it to get some fresh results.

**Comment by dmsimard** [comment]
I can try and investigate later but in the meantime the job returned successfully: https://ansible.softwarefactory-project.io/zuul/build/50e09cec38da4fafbc5dc57136d8f1fe

**Comment by hille721** [comment]
Don't get me wrong. Don't see this as a bug or problem. We need to set REQUEST_CA_BUNDLE not only for Ara but for all our Python tools in which the requests library is used and there we have a lot. Thus not a problem for me and that's why I won't spend time for further investigation.

**Comment by hille721** [comment]
And just to clarify, we don't set any certificate / ca-bundle in the Ara config, but ONLY the REQUEST_CA_BUNDLE environment variable

**Comment by dmsimard** [comment]
Yes, it's a common enough issue that I have seen with python things before but I had not noticed it with ara (yet?).

I suppose since this is a known issue we should add a little something to the troubleshooting docs: https://github.com/ansible-community/ara/blob/master/doc/source/troubleshooting.rst

I'll leave this issue opened in the meantime.
